src/CommandLineUtils/CommandLineApplicationExtensions.cs

```diff
@@ -43,21 +52,58 @@ public static void OnExecute(this CommandLineApplication app, Action action)
         /// <summary>
         /// Finds <see cref="AssemblyInformationalVersionAttribute"/> on <paramref name="assembly"/> and uses that
         /// to set <see cref="CommandLineApplication.OptionVersion"/>.
+        /// <para>
+        /// Uses the Version that is part of the <see cref="AssemblyName"/> of the specified assembly
+        /// if no <see cref="AssemblyInformationalVersionAttribute"/> is applied.
+        /// </para>
         /// </summary>
         /// <param name="app"></param>
         /// <param name="assembly"></param>
-        public static void VersionOptionFromAssemblyAttributes(this CommandLineApplication app, Assembly assembly)
-            => app.VersionOption("--version", GetInformationalVersion(assembly));
+        /// <exception cref="ArgumentNullException">Either <paramref name="app"/> or <paramref name="assembly"/> is <c>null</c>.</exception>
+        public static CommandOption VersionOptionFromAssemblyAttributes(this CommandLineApplication app, Assembly assembly)
+            => VersionOptionFromAssemblyAttributes(app, "--version", assembly);
 
-        private static string GetInformationalVersion(Assembly assembly)
+        /// <summary>
+        /// Finds <see cref="AssemblyInformationalVersionAttribute"/> on <paramref name="assembly"/> and uses that
+        /// to set <see cref="CommandLineApplication.OptionVersion"/>.
+        /// <para>
+        /// Uses the Version that is part of the <see cref="AssemblyName"/> of the specified assembly
+        /// if no <see cref="AssemblyInformationalVersionAttribute"/> is applied.
+        /// </para>
+        /// </summary>
+        /// <param name="app"></param>
+        /// <param name="template"></param>
+        /// <param name="assembly"></param>
+        /// <exception cref="ArgumentNullException">Either <paramref name="app"/> or <paramref name="assembly"/> is <c>null</c>.</exception>
+        public static CommandOption VersionOptionFromAssemblyAttributes(CommandLineApplication app, string template, Assembly assembly)
```

Comment:
I'm okay adding the overload, but let's leave the current behavior of this extension method. I use it in my own apps and don't want this to change.